readme.md

```diff
@@ -71,7 +71,11 @@ There are two parts to creating an Application Link. The first is done in Jenkin
 To register a consumer: 
 1. In Jenkins go to **Jenkins** > **Manage Jenkins** > **Manage Bitbucket Server consumers**. 
 2. Select **Register new consumer**. 
-3. Enter consumer details. 
+3. Enter these details: 
+   - Consumer key: A unique identifier for your Bitbucket instance. 
```

Comment:
I think its important to make sure the user knows that they make these up, they don't have to look for it somewhere in Bitbucket